sources/light_client.move

```diff
@@ -426,9 +446,10 @@ public entry fun insert_headers(c: &mut LightClient, raw_headers: vector<vector<
     let first_header = new_block_header(raw_headers[0]);
     let latest_block_hash = c.latest_block().header().block_hash();
 
+    let mut fork_created = false;
```

Comment:
I am not sure about the name here. How about `is_forked`?

Reply:
Good name! Updated!